src/BulletSoftBody/btCGProjection.h

```diff
@@ -0,0 +1,149 @@
+//  btCGProjection.h
```

Comment:
Please add Zlib copyright header, with Google Inc. 2019
See example top of https://github.com/bulletphysics/bullet3/blob/500264ca239d3fdf69162c75c5b40ee04baae458/examples/RoboticsLearning/GripperGraspExample.h